crates/amdgpu_top_gui/src/lib.rs

```diff
@@ -128,6 +128,7 @@ pub fn run(
 
     let options = eframe::NativeOptions {
         initial_window_size: Some(egui::vec2(1080.0, 840.0)),
+        app_id: Some("amdgpu_top".to_string()),
```

Comment:
`title` is already used for `app_id`.

> If you don’t set [Self::app_id](https://docs.rs/eframe/0.22.0/eframe/struct.NativeOptions.html#structfield.app_id), the title argument to [crate::run_native](https://docs.rs/eframe/0.22.0/eframe/fn.run_native.html) will be used instead.

https://docs.rs/eframe/0.22.0/eframe/struct.NativeOptions.html#structfield.app_id

Reply:
which is pretty weird, as removing that line but leaving eframe at 0.22.0 with the wayland feature results in no app_id

Reply:
found the culprit I think, gonna make a PR to eframe

Reply:
now we wait https://github.com/emilk/egui/pull/3107